src/Tools/dotnet-user-jwts/src/Helpers/SerializerOptions.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json;
+
+namespace Microsoft.AspNetCore.Authentication.JwtBearer.Tools;
+
+internal static class SerializerOptions
```

Comment:
Happy to rename this if there's a better more-specific name.

Reply:
What about "JwtSerializerOptions"?

Reply:
(I don't love mine either but the good name is already taken by the type 😆)